core/src/credential.rs

```diff
@@ -0,0 +1,97 @@
+
+#![allow(unused_variables)]
+#![allow(dead_code)]
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+type VCContext = [String];
```

Comment:
should this be `[&'static str]` instead?

Reply:
This is going to be changed as whole so I don't think we'll be keeping it

Reply:
Yes, [&'static str] is better here.